bee-storage/bee-storage/src/backend.rs

```diff
@@ -21,6 +26,12 @@ pub trait StorageBackend: Send + Sized + Sync + 'static {
     /// Shutdowns the backend.
     async fn shutdown(self) -> Result<(), Self::Error>;
 
-    /// Size of the database in bytes. Not all backends may be able to provide it, hence the option.
+    /// Returns the size of the database in bytes. Not all backends may be able to provide it, hence the option.
     async fn size(&self) -> Result<Option<usize>, Self::Error>;
+
+    /// Returns the health status of the database.
```

Comment:
Maybe a bit finicky, but it would probably be good to repeat the "Not all backends may be able to provide it –" clause here, for the sake of the generated docs (in case someone isn't looking directly at the code to see the function above).

Reply:
Done